lib/shared-dam-app/package.json

```diff
@@ -9,7 +9,7 @@
     "@babel/plugin-transform-runtime": "7.8.3",
     "@babel/preset-env": "7.8.4",
     "@babel/preset-react": "7.8.3",
-    "@contentful/contentful-extension-scripts": "^0.19.0",
+    "@contentful/contentful-extension-scripts": "^0.15.1",
```

Comment:
How about reverting only forma packages?

Reply:
Yep could do that.